src/sentry/static/sentry/app/options.jsx

```diff
@@ -135,7 +135,7 @@ const definitions = [
   }
 ];
 
-const definitionsMap = _.indexBy(definitions, 'key');
+const definitionsMap = _.keyBy(definitions, def => def.key);
```

Comment:
So, that's a shorthand, and I need to include more code to support that. I actually went out of my way to change this.